src/libsync/propagatorjobs.cpp

```diff
@@ -159,7 +162,7 @@ void PropagateLocalMkdir::start()
     }
 
     if( Utility::fsCasePreserving() && propagator()->localFileNameClash(_item->_file ) ) {
-        qDebug() << "WARN: new folder to create locally already exists!";
+        qCDebug(lcPropagateLocalMkdir) << "WARN: new folder to create locally already exists!";
```

Comment:
... ^^ maybe also with filename if this one is warning but the one above with the filename is only debug?